src/Bind.php

```diff
@@ -182,4 +183,16 @@ public function setBound(DependencyInterface $bound)
     {
         $this->bound = $bound;
     }
+
+    /**
+     * @param string $interface
+     *
+     * @return bool
+     */
+    private function hasNotRegistered($interface)
+    {
+        $hasNotRegistered = ! isset($this->container->getContainer()[$interface . '-' . NAme::ANY]);
```

Comment:
There is a type on camel case `NAme::ANY`, and also you probably can return this line directly.